lib/Stats.js

```diff
@@ -576,6 +593,16 @@ class Stats {
 			});
 			table(t, "rrrlll");
```

Comment:
Oh this seems to work already, due to this whole block is encapsulated with:
```javascript
if(obj.assets && obj.assets.length > 0) {
     ...
}
```
Tested it local with filterAssets: /\.*/